python/ray/tune/tune.py

```diff
@@ -553,6 +556,9 @@ def sigint_handler(sig, frame):
         if trial.status != Trial.TERMINATED:
             incomplete_trials += [trial]
 
+    # Calls on_experiment_end on callbacks
+    runner.end_experiment_callbacks()
+
```

Comment:
This is a weird location for this call - move before `tune_taken`? (Or consolidate, see comment above)

Reply:
Moved